packages/react-dom/src/test-utils/ReactTestUtilsAct.js

```diff
@@ -103,9 +103,11 @@ function act(callback: () => Thenable) {
             if (called === false) {
               warningWithoutStack(
                 null,
-                'You called act(async () => ...) without await. ' +
+                'You called act(async () => ...) without awaiting its result. ' +
                   'This could lead to unexpected testing behaviour, interleaving multiple act ' +
-                  'calls and mixing their scopes. You should - await act(async () => ...);',
+                  'calls and mixing their scopes. You should await asynchronous act() ' +
+                  'calls, like so -\n' +
```

Comment:
We don't really use hyphens at the end of a sentence like this. Let's replace ", like so -" with ":"?